apps/studio/components/interfaces/HomeNew/GettingStarted/GettingStartedSection.tsx

```diff
@@ -0,0 +1,563 @@
+            value={selectedFramework}
+            onChange={setSelectedFramework}
+            items={FRAMEWORKS}
+          />
+        ),
+      },
+      {
+        label: 'Connect',
+        variant: 'default',
+        onClick: openConnect,
+      },
+    ],
+    [openConnect, selectedFramework]
+  )
+
+  const codeSteps: GettingStartedStep[] = useMemo(
+    () => [
+      {
+        key: 'install-cli',
+        status: 'incomplete',
+        title: 'Install the Supabase CLI',
+        icon: <Code strokeWidth={1} className="text-foreground-muted" size={16} />,
+        description:
+          'To get started, install the Supabase CLI to manage your project locally, handle migrations, and seed data.',
+        actions: [
+          {
+            label: 'Install via npm',
+            component: (
+              <CodeBlock className="w-full text-xs p-3 !bg" language="bash">
+                npm install supabase --save-dev
+              </CodeBlock>
+            ),
+          },
+        ],
+      },
+      {
+        key: 'design-db',
+        status: tablesCount > 0 ? 'complete' : 'incomplete',
+        title: 'Design your database schema',
+        icon: <Database strokeWidth={1} className="text-foreground-muted" size={16} />,
+        description: 'Next, create a schema file that defines the structure of your database.',
+        actions: [
+          {
+            label: 'Create schema file',
+            href: 'https://supabase.com/docs/guides/local-development/declarative-database-schemas',
+            variant: 'default',
+          },
+          {
+            label: 'Generate it',
+            variant: 'default',
+            icon: <AiIconAnimation size={14} />,
+            onClick: () =>
+              openAiChat(
+                'Design my database',
+                'Help me create a schema file for my database. We will be using Supabase declarative schemas which you can learn about by searching docs for declarative schema.'
+              ),
+          },
+        ],
+      },
+      {
+        key: 'add-data',
+        status: 'incomplete',
+        title: 'Seed your database with data',
+        icon: <Table strokeWidth={1} className="text-foreground-muted" size={16} />,
+        description: 'Now, create a seed file to populate your database with initial data.',
+        actions: [
+          {
+            label: 'Create a seed file',
+            href: 'https://supabase.com/docs/guides/local-development/seeding-your-database',
+            variant: 'default',
+          },
+          {
+            label: 'Generate data',
+            variant: 'default',
+            icon: <AiIconAnimation size={14} />,
+            onClick: () =>
+              openAiChat(
+                'Generate seed data',
+                'Generate SQL INSERT statements for realistic seed data that I can run via the Supabase CLI.'
+              ),
+          },
+        ],
+      },
+      {
+        key: 'add-rls-policies',
```

Comment:
Again, this step might be confusing because of local/hosted mix.

We've been assuming that the user is working locally all along but 'Create a migration file' suddenly gets them to create a policy in hosted dashboard